src/network/networkctl.c

```diff
@@ -1631,10 +1671,10 @@ static int link_status_one(
                            TABLE_EMPTY,
                            TABLE_STRING, "Link File:",
                            TABLE_SET_ALIGN_PERCENT, 100,
-                           TABLE_STRING, strna(link),
+                           TABLE_STRV, link_dropins ?: STRV_MAKE("n/a"),
                            TABLE_EMPTY,
                            TABLE_STRING, "Network File:",
-                           TABLE_STRING, strna(network),
+                           TABLE_STRV, network_dropins ?: STRV_MAKE("n/a"),
```

Comment:
I'm not a big fan of "n/a" in such contexts. It can mean many different things depending on context, and it's also visually heavy. In particular, if there is no network file or dropins, we should just say "(none)". But this PR doesn't materially change this, so it's fine to leave for another time.